util/src/kvdb.rs

```diff
@@ -151,7 +197,29 @@ pub trait KeyValueDB: Sync + Send {
 	fn transaction(&self) -> DBTransaction { DBTransaction::new() }
 
 	/// Get a value by key.
-	fn get(&self, col: Option<u32>, key: &[u8]) -> Result<Option<DBValue>, String>;
+	fn get(&self, col: Option<u32>, key: &[u8]) -> Result<Option<DBValue>, String> {
+		let mut o_func = Some(DBValue::from_slice);
+		let mut output = None;
+
+		{
+			let mut wrapper = |key: &[u8]| { output = Some((o_func.take().unwrap())(key)); };
```

Comment:
Yeah, this was a copy-paste thing because originally I tried to use `get_with`. Good catch.